src/backend/distributed/sql/udfs/time_partitions/latest.sql

```diff
@@ -0,0 +1,18 @@
+CREATE VIEW citus.time_partitions AS
+SELECT partrelid AS "Parent Table", attname AS "Parttiion Column", relid AS "Partition", lower_bound AS "From", upper_bound AS "To", amname AS "Access Method"
```

Comment:
Using identifiers that require quoting seems annoying. Do we already do that other places?

Reply:
you're probably right.

We have a view called public.citus_tables which "Uses Quoting" to closely mimic \d and we don't expect it to be that interesting to query, and I followed the same format here.

For this view it might be nicer to stick to lower_bound, upper_bound, ..., since we can then do:
```sql
SELECT alter_table_set_access_method(partition, 'columnar') FROM time_partitions WHERE upper_bound < now() - interval '7 days';
```